### PR TITLE
Making time table viz scrollable

### LINKED ISSUE #3802: Time Series Table can't be scrollable
Make sure these boxes are checked before submitting your issue - thank you!

- [x] I have checked the superset logs for python stacktraces and included it here as text if any
- [x] I have reproduced the issue with at least the latest released version of superset
- [x] I have checked the issue tracker for the same issue and I haven't found one similar

### Superset version
0.20.5

### Expected results
Time Series Table should be scrollable if there is a large number of group-by dimensions.

### Actual results
Unscrollable.

### Steps to reproduce
Make a Time Series Table with a large number of group-by dimensions.

snapshot

### PR DESCRIPTION
**Pull request by michellethomas**
Previously time table viz didn't scroll when given a large number of results.

Fixes https://github.com/apache/incubator-superset/issues/3802

### CODE REVIEW

File: superset/assets/visualizations/time_table.jsx [hunk 1]
```diff
@@ -36,6 +36,7 @@ FormattedNumber.propTypes = {
 
 function viz(slice, payload) {
   slice.container.css('height', slice.height());
+  slice.container.css('overflow', 'scroll');
```
Comment (graceguo-supercat) [inline superset/assets/visualizations/time_table.jsx]: why not set this style in css file, the container has a class...

**Review by graceguo-supercat — COMMENTED** [review]
Is there place where we change slice.

**Comment by michellethomas** [comment]
I'm not sure what the padding: 10px was set for, but it didn't seem to be getting applied correctly and it looks like too much padding.

**Review by graceguo-supercat — APPROVED** [review]
LGTM

**Pull request merged**